APIBacklog/documentation/SupportingDocuments/API proposals/IMEI Fraud

```diff
@@ -0,0 +1,14 @@
+IME Fraud API submission
+
+| **Field** | Description |
+| ---- | ----- |
+| API name | IMEI Fraud|
+| API owner | MTN - Waseem Amra |
+| Initial API Contributors | MTN - Waseem Amra | 
+| API summary | This API allows for the API Consumer to understand if fraud has been associated to the device. Fraud can range from lost, suspended, stolen, blacklisted. |
+| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud logs are associated with the device. </em>.
```

Comment:
```suggestion
| Technical viability | This API uses IMEI info, usually stored by the MNO, to determine whether any fraud markers are associated with the device. </em>.
```

Reply:
@wwAMRA if would expect that you either commit the change or comment why the change is not appropriate